tox.ini

```diff
@@ -22,3 +25,8 @@ commands =
     generate_validation: python setup.py generate_validation
     # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
     ansible_syntax: ansible-playbook --syntax-check playbooks/byo/config.yml
+
+    # Unset GOPATH because tests use relative imports. This should be removed if
+    # we require openshift-ansible to live in a Go work space and use absolute
+    # imports in tests (desirable).
+    integration: env -u GOPATH go test -v ./test/integration/...
```

Comment:
If I'm reading the tox docs correctly, the GOPATH env var shouldn't be available in the test environment with the default config. Outside of a handful of whitelisted vars, you should have to specify which env cars you want to make available using `passenv` or by explicitly defining them with `setenv`.